exec/java-exec/src/test/java/org/apache/drill/exec/store/log/TestLogReader.java

```diff
@@ -781,9 +781,19 @@ public void testPluginSerialization() throws IOException {
 
   @Test
   public void testFirewallSchema() throws RpcException {
-    String sql = "SELECT * FROM cp.`regex/firewall.ssdlog`";
+    String sql = "SELECT * FROM cp.`regex/firewall.ssdlog` limit 0";
     RowSet result = client.queryBuilder().sql(sql).rowSet();
-    result.print();
+    TupleMetadata expectedSchema = new SchemaBuilder()
```

Comment:
Another change you might consider is renaming either this, or the other `SchemaBuilder`. Since this is the one we want to use moving forward, maybe rename the other to `BatchSchemaBuilder` so that future folks don't have to figure out the correct import when the reference the class.

Reply:
I have reverted this change since it caused a lot of tests failures. I'll create a Jira to replace usage of another `SchemaBuilder` with `BatchSchemaBuilder`.

Reply:
Thanks for trying to fix this. Doing the change as another PR sounds good.

Reply:
I have created https://issues.apache.org/jira/browse/DRILL-7354 to address this change.